### PR TITLE
#DP-50 (docs): Improve readme file

### PR DESCRIPTION
**Pull request by feyton**
### What the PR does:
The PR add information about the project to our base README file to give people an overview

### Description of tasks to be completed:
- [x] Running instructions are added
- [x] Important links (Figma) are added

### How can this be manually tested
Open this [Link to view the new version of README file](https://github.com/atlp-rwanda/atlp-pulse-fn/tree/docs-readme-update-dp-50)

### Any additional context
In the pre-commit script, I was unable to commit due to the fact that the `lint` script was checking for all files including `webpack.config.js`. I rearranged the order to exclude `configurations` related files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# atlp-pulse-fn
\ No newline at end of file
+#### atlp-pulse-fn
+
+## Introduction
+
+Pulse is a platform designed to handle ratings for the companies in the **Edu-tech** industries with the its first paying customer being **Andela**. It is currently under development using modern web technologies that prioritize speed and security. This repo holds the codebase for the frontend part of the platform which uses **Reactjs** javascript framework for build modern UIs.
```
Comment (samyshyaka) [inline README.md:5]: It is Ed-tech, not Edu-tech

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
-# atlp-pulse-fn
\ No newline at end of file
+#### atlp-pulse-fn
```
Comment (samyshyaka) [inline README.md:1]: Here, you can use the name of the product as the title

**Review by samyshyaka — CHANGES_REQUESTED** [review]
Good job @feyton, attend to the feedback below.

**Review by samyshyaka — APPROVED** [review]

**Review by Julesbi — APPROVED** [review]

**Review by ceelogre — APPROVED** [review]

**Comment by Julesbi** [comment]
Resolve conflicts and get merged

**Pull request merged**